OneCommunityGlobal/HGNRest
# Samarth B Job CC feature

# Description
Create a UI for an Owner to add to the CC options for specific posting categories, jobs, and/or ALL positions - Create a modal or management dashboard for the owner to view and manage the CC lists.
Should show whomever is on this list
Make it so that the CC list checks to make sure the person is active, and removes them from the list if they aren’t
Have CC functionality to add other people to CC so people who are hiring for a particular position can get a CC of the application details
Allow the owner to assign CC permissions: At the category level (example all ads under “Software Engineer”), For individual ads within a category (example only Front-End Engineer ads).


## Related PRS (if any):
To test this backend PR you need to checkout the [#2924](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2924) frontend PR.
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. run backend and then follow frontend instructions

## Screenshots or videos of changes:

## Note:
Include the information the reviewers need to know.
